### ISSUE TITLE
No add-hline/add-vline/etc. in plot

### ISSUE THREAD

**Issue by WeetHet** [johannes-wolf/cetz#284]
```typst
#import "@preview/cetz:0.1.2"

#box(stroke: 2pt + red, cetz.canvas({
  import cetz.draw: *
  import cetz.plot

  plot.plot(size: (1, 1),
    x-tick-step: none,
    y-tick-step: none,
  {
    plot.add-vline(0)
    plot.add-hline(0)
    plot.add(((0,0), (1, 0)))
  })
}))
```

fails to compile with error `module plot does not contain add-vline`

**Comment by johannes-wolf** [comment]
It has not been released yet; 0.1.2 does not contain said functions.

**Comment by WeetHet** [comment]
Alright, thanks
